### ISSUE TITLE
Server settings page

### ISSUE THREAD

**Issue by GioPan04** [jarnedemeulemeester/findroid#670]
**Is your feature request related to a problem? Please describe.**
The app works just fine, but having a server settings page, like the one in the web ui, wouldn't need the official jellyfin app to manage the server.
Of course this would need much work, but if the maintainers are okay with the feature I'm willing to help.

**Describe alternatives you've considered**
The only alternative would be to use the original app or the web ui, but that's the problem.

**Comment by UriSheinman** [comment]
I've seen this here a bunch of times (also mentioned a lot on discord), and I strongly disagree. Adding admin tools would just make it cluttered like the official app. They should stay separate, one for management and one for viewing.

**Comment by GioPan04** [comment]
I don't think that the admin tools are cluttering the official app, but at the end this are just personal preferences.
Also I don't see how would clutter the app, the option would be shown only to admin users and will be "hidden" in the settings.
At this point I have the official app only to add a new user or change a quick setting.

**Comment by Jcuhfehl** [comment]
I agree it wouldn't clutter the app. However, it's probably infeasible to implement this. If we don't include *every* setting, server admins would still need to keep the official app installed, so we wouldn't really add anything. Additionally, if jellyfin adds any new settings, we would have to manually add them to findroid, meaning the official app would be needed anyway.

Basically: why spend a lot of time implementing this if we can't replace the official app which will always do administration better

**Comment by AbandonedCart** [comment]
Are we intentionally overlooking that the server address you enter is the site of the Jellyfin server with its **integrated web UI**. Imagine clicking a single button and being whisked away to a WebView or even a generic browser Intent that opens the server to `{SERVER}/web/index.html#!/dashboard.html`.

**Comment by TheRedSpy15** [comment]
As someone who has foldering watching off on my server. And manually rescan folders outside of a scheduled scan in the morning

Simply having the option to trigger a folder rescan would be all the administrative tooling I'd need. Anything else I'd manage from home on my desktop/laptop anyways

**Comment by sannidhyaroy** [comment]
While I appreciate that this makes my admin tasks much more sleeker, focus on this right now isn't feasible. Admin tools are niche, primarily for administrators, so prioritizing them over fixing bugs or enhancing user experience for media consumption may not be ideal. After all, Jellyfin is a media server, meant for enjoying your content, so admin tools, though necessary, serve a smaller role in this goal. Administrators can access the admin dashboard via the Jellyfin Web interface from their browser when needed, and the official Jellyfin app essentially mirrors this functionality as it's just a wrapper around it. Therefore, enhancing the core viewing experience should remain the primary focus of this project. I'm rather excited for Jellyfin Vue to bring admin tools.

**Comment by AbandonedCart** [comment]
Someone looking to gain the functionality without having to write a whole UI may want to consider [Custom Tabs](https://developer.chrome.com/docs/android/custom-tabs). The base implementation isn’t incredibly advanced and it it is a decent compromise between maintaining the native feel and still just displaying the server UI.

**Comment by sannidhyaroy** [comment]
> Someone looking to gain the functionality without having to write a whole UI may want to consider [Custom Tabs](https://developer.chrome.com/docs/android/custom-tabs). The base implementation isn’t incredibly advanced and it it is a decent compromise between maintaining the native feel and still just displaying the server UI.

That's a decent compromise. I agree with you there.

**Comment by hai265** [comment]
Hi everyone! I've posted a PR that launches the admin dashboard in an external browser (through Custom Tabs)
https://github.com/jarnedemeulemeester/findroid/pull/1174
